packages/mcl/src/src/mcl/commands/host_info.d

```diff
@@ -64,7 +67,7 @@ string[string] getProcInfo(string fileOrData, bool file = true)
 
 export void host_info()
 {
-    const params = parseEnv!Params;
+    params = parseEnv!Params;
```

Comment:
This should remain a local `const` variable. If necessary add parameters to the functions that need to access it.